src/analysis/normed/group/pointwise.lean

```diff
@@ -55,6 +55,18 @@ eq_of_forall_le_iff $ λ r, by simp_rw [le_inf_edist, ←image_inv, ball_image_i
 lemma inf_edist_inv_inv (x : E) (s : set E) : inf_edist x⁻¹ s⁻¹ = inf_edist x s :=
 by rw [inf_edist_inv, inv_inv]
 
+@[to_additive] lemma diam_mul_le (x y : set E) :
+  emetric.diam (x * y) ≤ emetric.diam x + emetric.diam y :=
```

Comment:
I couldn't work out whether the `metric.diam` version of this is true; either way, I don't need even this result, it just seemed like a natural one to add.

Reply:
Ahah, so I thought!

Reply:
`iso_mul_action` is `has_isometric_smul`

Reply:
I've ejected this from this PR, as I didn't need it anyway. Feel free to push whatever to #19028.